Tests.Template/integration_template.ps1

```diff
@@ -17,17 +17,18 @@ $Global:DSCModuleName      = 'x<ModuleName>' # Example xNetworking
 $Global:DSCResourceName    = 'MSFT_x<ResourceName>' # Example MSFT_xFirewall
 # /TODO
 
-#region HEADER
-if ( (-not (Test-Path -Path '.\DSCResource.Tests\')) -or `
-     (-not (Test-Path -Path '.\DSCResource.Tests\TestHelper.psm1')) )
+#region HEADER (V2)
```

Comment:
I think we can stick with HEADER, did you have any special use for adding V2 here?